utils/rsync-nbsd.sh

```
@@ -0,0 +1,9 @@
+TDNS_SRC=/Users/johani/src/git/tdns
+TDNS_NBSD_DIR=root@172.16.1.37:/tmp/
+
+if [ -n "$$TDNS_NBSD_DIR" ]; then
```

Shellcheck (0.10.0)

[error] Argument to -n is always true due to literal strings.

(SC2157)